src/designs/pure/mobile.css

```diff
@@ -9,3 +9,10 @@
 .page {
     overflow: visible;
 }
+
+#navbarLeft .navbar{
+  background-color: #000000;
+  -webkit-box-shadow: 1px 0px 16px gray;
+  -moz-box-shadow: 1px 0px 16px gray;
+  box-shadow: 1px 0px 16px gray;
+}
```

Comment:
No, I just added it to bettet show the overlapping navbar. I will remove it before a merge.